changelog.txt

```diff
@@ -26,7 +30,6 @@
 **WooCommerce Admin 1.5.0**
 * Enhancement - Add eWAY to Payment Setup for AU/NZ Stores. #4947
 * Fix - Use clipRule and fillRule props. #4889, part of #4864
-* Tweak: Remove the Composite products option from the onboarding wizard #4703
```

Comment:
This is to match the change in https://github.com/woocommerce/woocommerce-admin/pull/4995